quantized_mesh_tile/topology.py

```diff
@@ -161,7 +164,7 @@ def _loadGeometry(self, geometrySpec):
         """
         A private method to convert a (E)WKB or (E)WKT to a Shapely geometry.
         """
-        if geometrySpec.startswith('POLYGON Z'):
+        if type(geometrySpec) is str and geometrySpec.startswith('POLYGON Z'):
```

Comment:
If type of geometrySpec is bytes we will have an exception here. So first check type then content.